website/source/docs/providers/aws/r/elastic_transcoder_pipeline.html.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: "aws"
+page_title: "AWS: aws_elastictranscoder_pipeline"
+sidebar_current: "docs-aws-resource-elastic-transcoder"
+description: |-
+  Provides an Elastic Transcoder pipeline resource.
+---
+
+# aws\_elastictranscoder\_pipeline
+
+Provides an Elastic Transcoder pipeline resource.
+
+## Example Usage
+
+### Elastic Transcoder Pipeline
+
+```
+resource "aws_elastictranscoder_pipeline" "bar" {
+  input_bucket = "${aws_s3_bucket.input_bucket.bucket}"
+  name         = "aws_elastictranscoder_pipeline_tf_test_"
+  role         = "${aws_iam_role.test_role.arn}"
+
+  content_config = {
+    bucket        = "${aws_s3_bucket.content_bucket.bucket}"
+    storage_class = "Standard"
+  }
+
+  thumbnail_config = {
+    bucket        = "${aws_s3_bucket.thumb_bucket.bucket}"
+    storage_class = "Standard"
+  }
+}
+```
+
+## Argument Reference
+
+See ["Create Pipeline"](http://docs.aws.amazon.com/elastictranscoder/latest/developerguide/create-pipeline.html) in the AWS docs for reference. 
+
+The following arguments are supported:
+
+* `aws_kms_key_arn` - (Optional) The AWS Key Management Service (AWS KMS) key that you want to use with this pipeline.
+* `content_config` - (Optional) The ContentConfig object specifies information about the Amazon S3 bucket in which you want Elastic Transcoder to save transcoded files and playlists. (documented below)
+* `content_config_permissions` - (Optional) The permissions for the `content_config` object. (documented below)
+* `input_bucket` - (Required) The Amazon S3 bucket in which you saved the media files that you want to transcode and the graphics that you want to use as watermarks.
+* `name` - (Optional) The name of the pipeline. Maximum 40 characters
+* `notifications` - The Amazon Simple Notification Service (Amazon SNS) topic that you want to notify to report job status. (documented below)
+* `output_bucket` - The Amazon S3 bucket in which you want Elastic Transcoder to save the transcoded files.
+* `role` - The IAM Amazon Resource Name (ARN) for the role that you want Elastic Transcoder to use to transcode jobs for this pipeline.
+* `thumbnail_config` - The ThumbnailConfig object specifies information about the Amazon S3 bucket in which you want Elastic Transcoder to save thumbnail files. (documented below)
+* `thumbnail_config_permissions` - The permissions for the `thumbnail_config` object. (documented below)
```

Comment:
Looks like (Optional) and (Required) didn't get mentioned on these last few. Can you also add `Required, Forces new  resource` for `ForceNew` attrs? It's helpful for users to see that.

Someday these will alll be generated! 😄